serverjs/updatecards.js

```diff
@@ -821,6 +817,42 @@ async function updateCardbase(ratings = [], basePath = 'private', defaultPath =
   }
 
   winston.info('Finished cardbase update...');
+};
+
+const downloadFromS3 = async (basePath = 'private') => {
+  winston.info('Downloading files from S3...');
+
+  const s3 = new AWS.S3({
+    accessKeyId: process.env.AWS_ACCESS_KEY,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  });
+
+  await Promise.all(
+    Object.keys(carddb.fileToAttribute).map(async (file) => {
+      const res = await s3
+        .getObject({
+          Bucket: 'cubecobra',
+          Key: `cards/${file}`,
+        })
+        .promise();
+      await fs.writeFileSync(`${basePath}/${file}`, res.Body);
+    }),
+  );
+
+  winston.info('Finished downloading files from S3...');
+};
+
+async function updateCardbase(ratings = [], basePath = 'private', defaultPath = null, allPath = null) {
+  if (!fs.existsSync(basePath)) {
+    fs.mkdirSync(basePath);
+  }
+  winston.info('Updating cardbase, this might take a little while...');
+
+  if (process.env.USE_S3) {
```

Comment:
Do a string comparison with `'true'` here so defining it as "false" wouldn't run with S3.